templates/charms-architecture.html

```diff
@@ -0,0 +1,191 @@
+{% extends 'how-juju-works.html' %}
+
+{% block title %}Charms architecture{% endblock %}
+{% block meta_keywords %}Software operators{% endblock %}
+{% block meta_copydoc %}https://docs.google.com/document/d/1nMN5UlwGqI4C4GmYcKy4yRzd9ANUmsVyB1ZcKY7Wpro/edit#heading=h.4tjfso5hx6tw{% endblock meta_copydoc %}
+{% block page_class %}is-paper{% endblock %}
+
+{% set selected_tab='charms-architecture' %}
+
+{% block how_juju_works_content %}
+<section class="p-strip u-no-padding--bottom u-no-padding--top">
+  <div class="row">
+    <div class="col-6">
+      <h1 class="p-heading--2">What is a charm?</h1>
+    </div>
+    <div class="col-6">
+      <p class="p-heading--5">A charm is software that wraps an application and that contains all of the instructions necessary for deploying, configuring, operating an application on any cloud using Juju.</p>
+      <p>A charm generally contains the Juju unit agent, the operations (charm) code and the application itself (the workload). Each application has a number of deployed "units" and includes information about the services they offer or require. Charms scale with the application and the operations code is always right next to the workload it is driving.</p>
+      <p>Charms include deterministic logic that specifies what happens when specific events occur. Events can be triggered by an administrator (e.g. through the CLI), by other charms or the external environment.</p>
+      <p>The administrator interacts with a client (CLI, terraform or Jimm), which talks to the controller. The controller talks to the agent in the charm, which in turns invokes the charm code to manipulate the application.</p>
+      <p>Charms are currently of two kinds, depending on the target deployment substrate:</p>
+      <ol>
+        <li><strong>Machine charms: </strong>Charms made to deploy on a bare-metal server, virtual machine, or system container.</li>
+        <li><strong>Kubernetes charms: </strong>Charms built to deploy on Kubernetes.</li>
+      </ol>
+    </div>
+  </div>
+  <div class="u-fixed-width" style="padding-top: 2rem; padding-bottom: 1rem">
+    {{ image(
+        url="https://assets.ubuntu.com/v1/905f1670-what%20is%20a%20charm.svg",
+        alt="",
+        width="1500",
+        height="836",
+        hi_def=True,
+        attrs={"class": "u-full-width"}
+      ) | safe
+    }}
+  </div>
+  <div class="row">
+    <div class="u-fixed-width col-start-large-4" style="padding-top: 2rem;">
+      <hr class="p-rule--muted">
+    </div>
+    <div class="col-3 col-start-large-4">
+      <p class="p-heading--5">What are agents and workers</p>
+    </div>
+    <div class="col-9 col-start-large-7">
```

Comment:
Numbers don't add up, we have 12 columns. Column of size 9 can't start at column 7 (this would add to 16). It was probably meant to be `col-6 col-start-large-7`